### ISSUE TITLE
[Website update] Common top-navigation and footer

### ISSUE THREAD

**Issue by krisfreedain** [opensearch-project/project-website#1208]
**Is your feature request related to a problem? Please describe.**
Currently, the project-website repo & documentation-website repo contain duplicates of the top navigation menu, as well as the footer. This becomes a problem every time we update the menu on the project-website as duplicative work needs to happen on the documentation-website

This is also an issue where our forum (https://forum.opensearch.org/) has a different set of top-nav & footer. To have a common user experience across the different properties, the forum should have the same top-nav & footer as the project-website repo.

**Describe the solution you'd like**
All top-nav menus and footers should be common and be read directly from the project-website repo. This would mean changing the documentation-website repo to only pull it's theme and navigation from the project-website. This would also mean building a solution for the forum, running on Discourse, to pull in the common top-nav and footer from the project-website repo.

**Comment by hdhalter** [comment]
Thanks, Kris! We desperately need this. @KrooshalUX - is this on your radar to design?

**Comment by kgcreative** [comment]
@AMoo-Miki - Miki, is this technically feasible given our current stack today? Or is this something we'll need to continue to do provisionally across the different web properties?

**Comment by KrooshalUX** [comment]
@hdhalter yes, the shared nav UI was identified as a starting point in the discovery meeting for the content component library. @krisfreedain -  Lets do a follow up next week to discuss timing and prioritization, I'll include Carlos (unable to tag in GH for some reason)

**Comment by AMoo-Miki** [comment]
While a uniform header and footer is indeed possible for the website and docsite, I am not sure how customizing the forum header and footer are achieved.

**Comment by hdhalter** [comment]
@krisfreedain - Is there any way we can move up the priority level? It would be a huge usability improvement to the site. If there are designs being discussed, can I get a preview? Thanks.

The navigation is completely different when you move from other pages in the website:
<img width="1205" alt="image" src="https://user-images.githubusercontent.com/104463754/226068596-4a41bd0c-fde8-4d0c-a19a-657cd0cc3b75.png">

To the documentation page:
<img width="1208" alt="image" src="https://user-images.githubusercontent.com/104463754/226068646-d041693d-1aa7-4d5d-82e5-c4c1986a8696.png">

**Comment by KrooshalUX** [comment]
@hdhalter OSDS is launching a documentation website which is the testing ground for a new theme, page templates and components that will be shared with .org and Docs.

We are currently in the requirements gathering phase for OWL (OpenSearch Web Library, repo & project plan coming soon from OpenSearch Design System)/ @kolchfa-aws submitted her list today and there is another set coming from the .org side as well.  Page templates and the shared component library (OWL) should address this issue. 

As soon as I get both sets of requirements, I will be mashing them together to make a master list against my Design System documentation website requirements, and then we will kick off the meeting series for the project. I imagine we might close some issues in this repo in favor of ones in OWL to build these components, and the tasks here will be related to implementing them.

**Comment by hdhalter** [comment]
We may need to make some changes to the Documentation bar in the next couple of weeks to incorporate something we need to do on our side (adding a new, "separate" site for Data Prepper ). Should we go ahead and make our changes - and while we're at it update the nav menu? In any event, we'll want to review the requirements for the common header nav pane to ensure we know what is coming vs. what we may need to provide on a separate documentation menu.

**Comment by krisfreedain** [comment]
@nateynateynate - can we assign this to you? It should be a small change to the Doc site template to just reference the menu on the .org site - you up for that?

**Comment by hdhalter** [comment]
Hi @nateynateynate - we also need to make the footer common. They are not in sync at the moment. Until we can get the common page, can we get the navigation menu and the footer menus consistent? Thanks.
